mobile/android/Android_Studio_Setup.md

```diff
@@ -0,0 +1,18 @@
+## Setup Android Studio
+###Download/Install Android Studio
+To build our Android app we'll be using an IDE called Android Studio. An IDE is a programming environment that allows you to edit files, compile and run your code, and has other features built into it like debugging. Lets start by [Downloading Android Studio](http://developer.android.com/training/basics/firstapp/index.html) To get it to work you'll also need to install [Java](TODO:link).
+
+###Download SDK tools
+TODO: SDK stuff here or after? test fresh install studio on windows.
+
+###Setting up a New Project
+Once Android studio is set up we'll need to start a [New Project](http://developer.android.com/training/basics/firstapp/creating-project.html). Open Android studio and select 'Start a new Android Studio Project', use a Minimum SDK of API 16 (Jelly Bean), and when prompted to choose starting Activity, choose 'Empty Activity'
+Minimum SDK is the minimum version of the Android Operating System required to run your app. We chose 16 because most phones run that or newer, and it allows us to use new Android features.
+The starting Activity is the code that Android Studio will generate to start your project. Well explain what Activity and Fragment are in a bit.
```

Comment:
Rather than just introducing the term "Activity" it may be more approachable if it's phrased "Android Studio will generate some starting code to start off your project. This is the starting Activity. We'll get more into what this means in a bit." or smthng